cmd/flags.go

```diff
@@ -303,6 +304,14 @@ func processorFlags(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
 	return cmd
 }
 
+func memoFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().String(flagMemo, "", "a memo to include in relayed packets")
```

Comment:
Do we need highlight we always append a suffix like `| rly(v2.0.0)`?

Reply:
Let's offer a way to omit the memo. I think empty defaulting to the version is reasonable. "-" as a special case to omit the memo seems reasonably typical.

Reply:
Documented the behavior in the README and added the behavior to omit using `-`